cayennelpp/lpp_data.py

```diff
@@ -1,6 +1,11 @@
 from .lpp_type import get_lpp_type
 
-import logging
+try:
+    import logging
+except ModuleNotFoundError:
+    class logging:
+        def debug(*kargs):
```

Comment:
somehow the params need to be defined differently ...

Reply:
What do you want to change here?

Reply:
to be honest: not sure right now, but [codacy](https://app.codacy.com/app/smlng/pycayennelpp/pullRequest?prid=3145595) complains about it

Reply:
according to codacy it is missing the `self`, so try to change to the following `def debug(self, *args, **kwargs)`